library/lib.sh

```diff
@@ -81,28 +81,35 @@ rolesGetTests() {
 # Handle Ansible Vault encrypted variables
 rolesHandleVault() {
     local role_path=$1
-    local playbook_file=$2
-    local vault_pwd_file="$role_path/vault_pwd"
-    local vault_variables_file="$role_path/vars/vault-variables.yml"
-    local no_vault_file="$role_path/no-vault-variables.txt"
+    local playbook_file=$role_path/tests/$2
+    local vault_pwd_file=$role_path/tests/vault_pwd
+    local vault_variables_file=$role_path/tests/vars/vault-variables.yml
+    local no_vault_file=$role_path/no-vault-variables.txt
     local vault_play
 
-    if [ -f "$vault_pwd_file" ] && [ -f "$vault_variables_file" ]; then
+    if [ ! -f "$vault_pwd_file" ] || [ ! -f "$vault_variables_file" ]; then
+        rlLogInfo "Skipping vault variables because $vault_pwd_file and $vault_variables_file don't exist"
+        return
+    fi
+    if [ -f "$no_vault_file" ]; then
         if grep -q "^${playbook_file}\$" "$no_vault_file"; then
```

Comment:
The format of the no_vault_file is the basename of the file e.g. from the ha_cluster role:

```
tests_default.yml
```